addons/virtualcamera/TransformModifiers/Follow.gd

```diff
@@ -1,49 +1,49 @@
-tool
+@tool
 extends "res://addons/virtualcamera/TransformModifiers/TransformModifier.gd"
 # Follows target keeping set distance and offset.
 class_name Follow
 
 # Target to follow (settable from editor).
-export var target_path : NodePath
+@export var target_path : NodePath
 # Target to follow (settable runtime).
-var target : Spatial
+var target : Node3D
 
 # Offset to apply from target's position in global coordinates.
-export var offset : Vector3 = Vector3.ZERO
+@export var offset : Vector3 = Vector3.ZERO
 
 # Distance that Follow isn't allowed to go any closer to target.
-export(float, EXP, 0.0, 2048.0) var min_distance : float = 1.0
+@export_range(0.0, 2048.0, , "exp") var min_distance : float = 1.0 # (float, EXP, 0.0, 2048.0)
```

Comment:
Tried compiling this locally in Godot 4.2, and the "@export_range(0.0, 2048.0, , "exp") throws a few errors. Adjusting this to "@export_range(0.0, 2048.0)" fixes this. 

The same change applies to the @export_range calls below this one.

Reply:
Ah, I suppose something changed between 4.0 and 4.2 that broke those. Very well, I added a second commit with the suggested changes, the plugin seems to compile fine now.